Makefile

```diff
@@ -9,6 +9,7 @@ docs: ## Generate a documentation Docker container
 	docker run --rm \
 		-u $(shell id -u):$(shell id -g) \
 		-v $(shell pwd)/docs:/documents/kukur \
+		-v $(shell pwd)/docs/source/:/documents/source \
```

Comment:
I would think this is not needed, since `docs/source` is already mounted and relative links should work both inside and outside the container?